plugins/node/instrumentation-amqplib/src/amqplib.ts

```diff
@@ -74,6 +74,8 @@ import {
 } from './utils';
 import { VERSION } from './version';
 
+const supporterVersions = ['>=0.5.5 <1'];
```

Comment:
```suggestion
const supportedVersions = ['>=0.5.5 <1'];
```

Also the usages below.